Examples/Python/tests/test_examples.py

```diff
@@ -689,7 +689,7 @@ def test_refitting(tmp_path, detector_config, assert_root_hash):
     field = acts.ConstantBField(acts.Vector3(0, 0, 2 * u.T))
 
     seq = Sequencer(
-        events=10,
+        events=3,
```

Comment:
Why do we reduce the number of events?

Reply:
- https://github.com/acts-project/acts/pull/3733/commits/2308c6e65fc9a17f18f3685f2359c63fe31a83f8
- https://github.com/acts-project/acts/actions/runs/11347289035/job/31560494422

The GSF is throwing warnings with the correct digi file which causes the CI to fail. I can only circumvent this by avoiding the offending event.

Reply:
https://github.com/acts-project/acts/issues/3738